docs/DeveloperGuide.md

```diff
@@ -335,15 +335,21 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given format is missing the field header sn/.
 
-    * 3a1. Warenager shows an error message.
+    * 3a1. Warenager shows an error message and tells user to use the proper format.
 
       Use case resumes at step 2.
 
-* 3b. The given serial number is not found.
+* 3b. All inputted serial numbers are not found.
```

Comment:
Should "inputted" be "input" instead?

Reply:
User inputted the number already hence past tense.